src/hooks/gqty.ts

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from 'lib/gqty';
+
+import { useSelectedCircle } from 'recoilState';
+
+export const useCurrentOrg = () => {
+  const { circleId } = useSelectedCircle();
```

Comment:
`useSelectedCircleId` would be better here, then it doesn't depend on state being loaded in recoil

Reply:
perhaps another similar recoil atom is added for organization and it can be independent of the selected circle?

Reply:
> perhaps another similar recoil atom is added for organization and it can be independent of the selected circle?

the way the UI is currently, this is what makes the most sense to me